tests/autoyast/pxe_boot.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2014 SUSE Linux Products GmbH
+# Copyright (C) 2015 SUSE Linux GmbH
```

Comment:
Does this test module ever get used at all? I cant find anywhere that calls it..